<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-RQ9AFM9" name="com.rest.api.authentications.BasicAuth_Test" tests="1" failures="0" timestamp="09 Oct 2020 12:00:20 GMT" time="3.132" errors="1">
  <testcase name="basic_digestive_Aut_API_test" time="3.132" classname="com.rest.api.authentications.BasicAuth_Test">
    <error type="io.restassured.path.xml.exception.XmlPathException" message="Failed to parse the XML document">
      <![CDATA[io.restassured.path.xml.exception.XmlPathException: Failed to parse the XML document
at io.restassured.path.xml.XmlPath$LazyXmlParser.invoke(XmlPath.java:1039)
at io.restassured.path.xml.XmlPath.getFromPath(XmlPath.java:423)
at io.restassured.path.xml.XmlPath.get(XmlPath.java:317)
at io.restassured.path.xml.XmlPath.getString(XmlPath.java:571)
at com.rest.api.authentications.BasicAuth_Test.basic_digestive_Aut_API_test(BasicAuth_Test.java:124)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 10; DOCTYPE is disallowed when the feature "http://apache.org/xml/features/disallow-doctype-decl" set to true.
at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1240)
at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:644)
at groovy.util.XmlSlurper.parse(XmlSlurper.java:201)
at groovy.util.XmlSlurper.parse(XmlSlurper.java:254)
at groovy.util.XmlSlurper.parseText(XmlSlurper.java:280)
at io.restassured.path.xml.XmlPath$1.method(XmlPath.java:840)
at io.restassured.path.xml.XmlPath$LazyXmlParser.invoke(XmlPath.java:1032)
... 29 more
]]>
    </error>
  </testcase> <!-- basic_digestive_Aut_API_test -->
</testsuite> <!-- com.rest.api.authentications.BasicAuth_Test -->
